arm/Microsoft.ContainerService/managedClusters/agentPools/readme.md

```diff
@@ -0,0 +1,70 @@
+# ContainerServiceManagedClustersAgentPools `[Microsoft.ContainerService/managedClusters/agentPools]`
+
+This module deploys an Agent Pool for a Container Service Managed Cluster
+
+## Resource types
+
+| Resource Type | API Version |
+| :-- | :-- |
+| `Microsoft.ContainerService/managedClusters/agentPools` | 2021-05-01 |
+
+## Parameters
+
```

Comment:
Will add this comment here so that is is not lost: Please also regenerate the parent readme. However, please make sure you first align the folder names. You called the folder correctly `agentpools` as per the ARM reference. The parameter in the parent bicep template should use the same parameter (instead of `additionalAgentPools`). It is currently more specific, but at the same time we need this alignment to properly auto-reference the child-resource readme.

Reply:
Regenerated both readme files using the script this time which corrected the parameter name consistency.